Source/control.cpp

```diff
@@ -1132,7 +1132,7 @@ void DrawTalkPan(const Surface &out)
 
 	x += 46;
 	int talkBtn = 0;
-	for (int i = 0; i < 4; i++) {
+	for (size_t i = 0; i < Players.size(); i++) {

```

Comment:
This is fixed to the number of frames in TalkButton, this change will leave to a crash if player size is incrased past 4.
```suggestion
	for (size_t i = 0; i < 4; i++) {
```

Reply:
Done